Lusas_Adapter/CRUD/Create/Loads/Loadcase.cs

```diff
@@ -51,6 +51,22 @@ private IFLoadcase CreateLoadcase(Loadcase loadcase)
             if (d_LusasData.existsLoadset(loadcase.Name))
             {
                 lusasLoadcase = (IFLoadcase)d_LusasData.getLoadset(loadcase.Name);
+                if (lusasLoadcase.getID() != loadcase.Number) 
```

Comment:
Do we need to update the number here? Because when we return it the `Loadcase`, there is going to be a conflict between the two `LusasId` and the `Number` of the Loadcase.

Reply:
Yes agreed that we should. Changes implemented so that the `loadcase.Number` get changed to match the lusas Loadcase number.